tests/e2e-tests/config/jest.setup.js

```diff
@@ -46,97 +45,48 @@ const OBSERVED_CONSOLE_MESSAGE_TYPES = {
- *
- * @return {Promise} Promise resolving once posts have been trashed.
- */
-async function trashExistingPosts() {
-	await switchUserToAdmin();
-	// Visit `/wp-admin/edit.php` so we can see a list of posts and delete them.
-	await visitAdminPage( 'edit.php' );
-
-	// If this selector doesn't exist there are no posts for us to delete.
-	const bulkSelector = await page.$( '#bulk-action-selector-top' );
-	if ( ! bulkSelector ) {
-		return;
-	}
-
-	// Select all posts.
-	await page.waitForSelector( '#cb-select-all-1' );
-	await page.click( '#cb-select-all-1' );
-	// Select the "bulk actions" > "trash" option.
-	await page.select( '#bulk-action-selector-top', 'trash' );
-	// Submit the form to send all draft/scheduled/published posts to the trash.
-	await page.click( '#doaction' );
-	await page.waitForXPath(
-		'//*[contains(@class, "updated notice")]/p[contains(text(), "moved to the Trash.")]'
-	);
-	await switchUserToTest();
-}
-
-/**
- * Navigates to the product listing screen and bulk-trashes any product which exist.
- *
- * @return {Promise} Promise resolving once products have been trashed.
- */
-async function trashExistingProducts() {
-	await switchUserToAdmin();
-	// Visit `/wp-admin/edit.php?post_type=product` so we can see a list of products and delete them.
-	await visitAdminPage( 'edit.php', 'post_type=product' );
-
-	// If this selector doesn't exist there are no products for us to delete.
-	const bulkSelector = await page.$( '#bulk-action-selector-top' );
-	if ( ! bulkSelector ) {
-		return;
-	}
-
-	// Select all products.
-	await page.waitForSelector( '#cb-select-all-1' );
-	await page.click( '#cb-select-all-1' );
-	// Select the "bulk actions" > "trash" option.
-	await page.select( '#bulk-action-selector-top', 'trash' );
-	// Submit the form to send all draft/scheduled/published posts to the trash.
-	await page.click( '#doaction' );
-	await page.waitForXPath(
-		'//*[contains(@class, "updated notice")]/p[contains(text(), "moved to the Trash.")]'
-	);
-	await switchUserToTest();
-}
-
 /**
  * Navigates to woocommerce import page and imports sample products.
  *
  * @return {Promise} Promise resolving once products have been imported.
  */
 async function importSampleProducts() {
 	await switchUserToAdmin();
-	// Visit Import Products page.
-	await visitAdminPage(
-		'edit.php',
-		'post_type=product&page=product_importer'
-	);
-	await page.click( 'a.woocommerce-importer-toggle-advanced-options' );
-	await page.focus( '#woocommerce-importer-file-url' );
-	// local path for sample data that is included with woo.
-	await page.keyboard.type(
-		'wp-content/plugins/woocommerce/sample-data/sample_products.csv'
-	);
-	await page.click( '.wc-actions .button-next' );
-	await page.waitForSelector( '.wc-importer-mapping-table' );
-	await page.select(
-		'.wc-importer-mapping-table tr:nth-child(29) select',
-		''
+	// Visit `/wp-admin/edit.php?post_type=product` so we can see a list of products and delete them.
```

Comment:
Is this comment still accurate?

Reply:
No.

Another point, changes in `tests/e2e-tests/config/jest.setup.js` are not done in this PR, they come from https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2544, this PR is based on it (I forget to mention that, fixed now), but I'm comparing this PR against master and not that PR (so that travis can run).

Reply:
Ah, that makes it hard to review the code in this PR. Might be a better idea to get #2544 reviewed and merged first so this PR's changes are clear.

So .. is that comment getting deleted?